src/main/java/com/semmle/jira/addon/config/init/WorkflowUtils.java

```diff
@@ -167,6 +194,26 @@ static void addLayoutToWorkflow(String workflowName, String layoutJson) {
     ofBizDelegator.createValue("OSPropertyText", textFields);
   }
 
+  private static void addFunction(ActionDescriptor transition, FunctionDescriptor function) {
+    @SuppressWarnings("unchecked")
+    List<FunctionDescriptor> functions = transition.getUnconditionalResult().getPostFunctions();
+    // remove existing functions with the same name
```

Comment:
You don't set a name on the function  so the entire `remove existing functions with the same name` block could be removed.

Reply:
I'm setting the name in line 89, at the beginning of `addDefaultPriorityToWorkflow`.

Reply:
Ah yes indeed.